chronos.py

```diff
@@ -0,0 +1,236 @@
+from __future__ import unicode_literals
+import requests
+
+from django.db import transaction
+
+from osf.models import ChronosJournal
+from osf.models import ChronosSubmission
+from osf.utils.workflows import ChronosSubmissionStatus
+from addons.osfstorage.models import OsfStorageFile
+
+
+class ChronosSerializer(object):
+
+    @classmethod
+    def serialize_manuscript(cls, journal_id, preprint, status=ChronosSubmissionStatus.DRAFT.value):
+        return {
+            'AUTHORS': [
+                cls.serialize_author(contrib)
+                for contrib in preprint.node.contributor_set.filter(visible=True).select_related('user')
+            ],
+            'MANUSCRIPT_FILES': [
+                cls.serialize_file(preprint, file_node)
+                for file_node in OsfStorageFile.objects.filter(node=preprint.node)
+            ],
+            'ABSTRACT': preprint.node.description,
+            'ARTICLE_TYPE': 'research-article',  # ??
+            'DOI': preprint.preprint_doi,
+            'MANUSCRIPT_TITLE': preprint.node.title,
+            'PROVIDER_MANUSCRIPT_ID': preprint._id,
+            'CHRONOS_JOURNAL_ID': journal_id,
+            'MANUSCRIPT_URL': preprint.url,
+
+            # 'CATEGORIES': '',
+            # 'CHRONOS_SUBMISSION_URL': '',
+            # 'CLASSIFICATIONS': '',
+            # 'INVOICES': '',
+            # 'JOURNAL_EISSN': '',
+            # 'JOURNAL_ISSN': '',
+            # 'JOURNAL_ISSUE': '',
+            # 'JOURNAL_NAME': '',
+            # 'JOURNAL_VOLUME': '',
+            # 'KEYWORDS': '',
+            # 'LICENSE': '',
+            # 'PMC_ID': '',
+            # 'PUBLISHER_JOURNAL_ID': '',
+            # 'PUBLISHER_NAME': '',
+            # 'RUNNING_HEAD': '',
```

Comment:
Are these all commented out because in a real request they'd be based on user input, do we just not care about them yet, or is it because of API specification issues?